<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IrcAccountConfigurationWidget</class>
 <widget class="QWidget" name="IrcAccountConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="ConfigurationWidget_">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="IrcUserProfle_">
      <attribute name="title">
       <string>User profile</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Real name:</string>
         </property>
         <property name="buddy">
          <cstring>RealName_</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="RealName_"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>User name:</string>
         </property>
         <property name="buddy">
          <cstring>UserName_</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="UserName_"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Nicks (one per line):</string>
         </property>
         <property name="buddy">
          <cstring>NickNames_</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="NickNames_">
         <property name="plainText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Default settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="defaultIrcServerLabel">
           <property name="text">
            <string>IRC server:</string>
           </property>
           <property name="buddy">
            <cstring>DefaultIrcServer_</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="DefaultIrcServer_"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="defaultPortLabel">
           <property name="text">
            <string>Port:</string>
           </property>
           <property name="buddy">
            <cstring>DefaultPort_</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="DefaultPort_">
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="defaultEncodingLabel">
           <property name="text">
            <string>Encoding:</string>
           </property>
           <property name="buddy">
            <cstring>DefaultEncoding_</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="DefaultEncoding_"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="defaultChannelLabel">
           <property name="text">
            <string>Channel:</string>
           </property>
           <property name="buddy">
            <cstring>DefaultChannel_</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="DefaultChannel_"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
